joint_trajectory_controller/include/joint_trajectory_controller/joint_trajectory_controller_impl.h

```diff
@@ -382,6 +382,11 @@ update(const ros::Time& time, const ros::Duration& period)
       return;
     }
 
+    // Get state error for current joint
+    state_joint_error_.position[0] = state_error_.position[i];
+    state_joint_error_.velocity[0] = state_error_.velocity[i];
+    state_joint_error_.acceleration[0] = 0.0;
```

Comment:
```suggestion
    state_joint_error_.acceleration[0] = state_joint_error_.acceleration[i];
```

we don't have to know at this point if this is set to 0.0